larq_zoo/utils.py

```diff
@@ -11,6 +11,27 @@
 ImagenetDataset = namedtuple("ImagenetDataset", ["input_shape", "num_classes"])
 
 
+def slash_join(*args):
+    return "/".join(arg.strip("/") for arg in args)
+
+
+def download_pretrained_model(model, version, file, file_hash, cache_dir=None):
+    root_url = "https://github.com/plumerai/larq-zoo/releases/download/"
+
+    url = slash_join(root_url, model + "-" + version, file)
```

Comment:
I looked into this, but `urlparse.urljoin` seems to be intended to parse proper urls rather than do easy concatenating (see e.g. [this](https://stackoverflow.com/a/1793282/3425529) SO question)